packages/picasso/picasso.0.4.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Ghiles Ziat <ghiles.ziat@epita.fr"
+authors: [
+  "Ghiles Ziat <ghiles.ziat@epita.fr"
+  "Matthieu Journault <Matthieu.journault@lip6.fr>"
+  ]
+homepage: "https://github.com/ghilesZ/picasso"
+bug-reports: "https://github.com/ghilesZ/picasso/issues"
+dev-repo: "git+https://github.com/ghilesZ/picasso"
+build: [
```

Comment:
Can you add a `license` field? You can look in https://spdx.org/licenses/ for the indentifiers to use

Reply:
Please let me know the license and we can update the PR and merge

Reply:
Ping @ghilesZ, if you let me know what license to use I can update the opam file and merge the PR

Reply:
Sry for the delay, anything will do, let's say LGPL

Reply:
Thanks!